### ISSUE TITLE
Crud failed issue by users

### ISSUE THREAD

**Issue by sufyanel** [bryntum/odoo-support#129]
https://forum.bryntum.com/viewtopic.php?f=58&t=26818&p=133810#p133810

**Comment by UrzaBlock** [comment]
@sufyanel can I ask if this is still relevant? As in, was it indeed 3rd party plugins? If no, what were the findings up to this point?
And what of the mobile error reported? 
Please let me know.

**Comment by thomaspaulb** [comment]
@UrzaBlock Can you reproduce the IOS error that he got way down in the conversation?

**Comment by UrzaBlock** [comment]
No, I can’t. But that might be due to a different error that occurs before I can make the final steps happen.

Once I click on Gantt View Pro, the following screen comes up, which I can’t get passed:

**Comment by UrzaBlock** [comment]
![Screen when clicked on gantt view pro](https://github.com/bryntum/odoo-support/assets/108005292/54d45665-8110-4cc0-89c1-82689b4206a8)
